config/install/editor.editor.comments.yml

```diff
@@ -29,10 +29,12 @@ settings:
       status: ''
     mentions:
       enable: 1
+      mentions_type: user
```

Comment:
I'm just curious, how user mentions are relevant to the block on the login form?

Reply:
Removed